README.rst

```diff
@@ -1,44 +1,66 @@
 Falcon |Docs| |Build Status| |codecov.io|
 =========================================
 
+We believe in using the best tool for the job. We also believe that the
+best tools are simple, reliable and versatile.
```

Comment:
👍 I think the Antoine quote is sufficient to convey Falcon's overarching design philosophy.